### PR TITLE
Adding trilium next, the community driven fork of trillium

### PR DESCRIPTION
**Pull request by lando786**

### CODE REVIEW

**Comment by lando786** [comment]
@nwithan8 thanks for feedback and approving. What are next steps to get this merged? Do I tag someone in particular or will someone check in on approved PRs?

**Comment by nwithan8** [comment]
Waiting for second review. Tagged @Roxedus , he's usually pretty active.

**Comment by Roxedus** [comment]
Fix the networking stuff the linter complains about

File: templates/triliumnext.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <Privileged>false</Privileged>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository/</Support>
+  <Project>https://github.com/TriliumNext/Notes</Project>
+  <Overview>TriliumNext Notes is an open-source, cross-platform hierarchical note taking application with focus on building large personal knowledge bases.</Overview>
+  <Category>Productivity: Tools:Utilities</Category>
+  <WebUI>http://[IP]:[PORT:8080]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/triliumnext.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/triliumnext.png</Icon>
+  <Description>TriliumNext Notes is a hierarchical note taking application with focus on building large personal knowledge bases.</Description>
+  <Networking>
```
Comment (nwithan8) [inline templates/triliumnext.xml:17]: Remove this tag, unneeded since `Network` is defined above

Comment (lando786) [reply templates/triliumnext.xml]: Removed, also spotted an empty Labels tag I missed the first time around, removed that as well.

**Pull request merged**